pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/JSONSchemaTest.java

```diff
@@ -380,16 +382,15 @@ public void testEncodeAndDecodeObject() throws JsonProcessingException {
     }
 
     @Test
-    public void testGetNativeSchema() throws SchemaValidationException {
+    public void testGetNativeSchema() throws IllegalAccessException {
         JSONSchema<PC> schema2 = JSONSchema.of(PC.class);
-        org.apache.avro.Schema avroSchema2 = (Schema) schema2.getNativeSchema().get();
+        org.apache.avro.Schema avroSchema2 =
+                (Schema) schema2.getNativeSchema().orElseThrow(IllegalAccessException::new);
```

Comment:
Is this change necessary?

Reply:
it was unsafe to call get on optional without checking it. hence I added this exception there

Reply:
and I fixed it as it was just a test class so should not cause any unexpected behaviour

Reply:
this is a test case, I am not sure it is worth to complicate it.
the test would fail anyway, we are only making the line longer IMHO

Reply:
I think that the possible "unsafe call to get" should be fine in this context (in test code).